src/renderer/actions/uninstall.js

```diff
@@ -14,32 +14,33 @@ import kill from "./utils/kill";
 import {showRestartNotice} from "./utils/notices";
 import doSanityCheck from "./utils/sanity";
 
-
 const DELETE_SHIM_PROGRESS = 85;
 const RESTART_DISCORD_PROGRESS = 100;
 
 
-async function deleteShims(paths) {
-    const progressPerLoop = (DELETE_SHIM_PROGRESS - progress.value) / paths.length;
-    for (const discordPath of paths) {
-        log("Removing " + discordPath);
+async function deleteShims(discordPaths) {
+    const progressPerLoop = (DELETE_SHIM_PROGRESS - progress.value) / discordPaths.length;
+    for (const discordPath of discordPaths) {
+        log(`Removing injection scripts from: ${discordPath}.`);
         const appPath = path.join(discordPath, "app");
         const indexFile = path.join(discordPath, "index.js");
         try {
-            if (process.platform === "win32" || process.platform === "darwin") {
+            if (process.platform === "win32" || process.platform === "darwin") {  // Windows and Mac.
                 if (await exists(appPath)) {
                     const error = await new Promise(r => rimraf(appPath, originalFs, r));
-                    if (error) throw error; // Throw instead because there are multiple throw points
+                    if (error) throw error;  // Re-throw the error to be caught by our universal handler.
```

Comment:
> Re-throw

It's not a re-throw because the promise resolves to this value.

Reply:
Ahh, good point! Will fix the comment.